bee-plugin/src/event.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Types for events.
+
+pub use crate::grpc::{DummyEvent, SillyEvent};
+
+use std::convert::TryFrom;
+
+/// Identifier for each event type/
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+#[repr(u8)]
+pub enum EventId {
```

Comment:
Actually I'm not sure if I want to remove them or not because otherwise the examples won't work.

Reply:
Just to be clear. Those aren't events that I made up for the examples themselves. I made them because there is no events at the moment :trollface:

For me the idea is that this `EventId` and anything in the `.proto` file can only change between versions. So once we have real events we'd have to update the examples.

Reply:
So this means, anyone who want to write a node plugin and run it with the node just need to edit the `.proto` file? And then it works without any changes to the code of the node?

Reply:
not exactily. It means that anyone who wants to write a plugin can use the `.proto` file as it is (once we have actual events to put in it).

My point is that we could leave the `.proto` file without events but then there would be no examples at all until we add events to the `.proto` file.

Reply:
OK I am using real events now that we are closer to have a network able to receive messages and we have a way to parse them.